### PR TITLE
Redesigned toggle button

### PR DESCRIPTION
**Pull request by Sahitiakella**

### CODE REVIEW

File: src/pages/DocsPage.tsx [hunk 1]
```diff
@@ -99,6 +103,8 @@ function DocsPage() {
               {/*   <span className="switchSun"> <span style={theme === 'light' ? { color: '#ffc045' } : { color: 'grey' }}>☀</span></span>
               <div className="switchMoon"> <span style={theme === 'light' ? { color: 'grey' } : { color: 'white' }}>☾</span></div> */}
               <label className="switch__label">
+                <FontAwesomeIcon icon={faSun} className="faSun" />
+                <FontAwesomeIcon icon={faMoon} className="faMoon" />
```
Comment (Niyousha99) [inline src/pages/DocsPage.tsx:107]: It looks like the chechbox is just covering one of the icons each time. You can see the sun icon peeking out of the checkbox: 
image

Let's these icons render conditionally depending on the theme, like this:
` <FontAwesomeIcon> { theme === "light" ? icon={faMoon} className="faMoon" : icon={faSun} className="faSun" } />`

**Review by Niyousha99 — CHANGES_REQUESTED** [review]
Now that there is enough contract between the toggle button icons anch background colors, let's remove the darker borders please.

**Review by Niyousha99 — APPROVED** [review]
Looks great, good job!

**Review by TakahiroFujitaSJB — APPROVED** [review]
Looks good to me, though I have no way to check it for mobile view.

**Pull request merged**